app/daemon.go

```diff
@@ -709,6 +711,20 @@ func (d *MenderShellDaemon) routeMessageShellResize(message *ws.ProtoMsg) error
 	return nil
 }
 
+func (d *MenderShellDaemon) routeMessagePongShell(message *ws.ProtoMsg) error {
+	var err error
+
+	s := session.MenderShellSessionGetById(message.Header.SessionID)
+	if s == nil {
+		err = session.ErrSessionNotFound
+		d.routeMessageResponse(nil, err)
```

Comment:
@tranchitella this is the place wher npr happens (Alf recent report)

Reply:
You are right, @merlin-northern
Fixed: https://github.com/mendersoftware/mender-connect/pull/9/commits/0a38b0a5277c273ff8a3c9d6bd132242abdd9dbd